src/query/src/analyze.rs

```diff
@@ -226,3 +234,61 @@ fn create_output_batch(
 
     builder.finish()
 }
+
+fn metrics_to_string(metrics: RecordBatchMetrics, format: AnalyzeFormat) -> String {
+    match format {
+        AnalyzeFormat::JSON => JsonMetrics::from_record_batch_metrics(metrics).to_string(),
+        _ => metrics.to_string(),
```

Comment:
List the variant explicitly:
```suggestion
        AnalyzeFormat::TEXT => metrics.to_string(),
```

Reply:
I mean to ignore it. There is another format like `graphviz`, which we are not likely to support shortly nor want to throw an error.

Reply:
Our code style prefers listing all variants in the match enumeration type if I don't get wrong.

Reply:
If the format doesn't support, let's return an error?